File-Formats/PDF/Working-with-OCR/Features.md

```diff
@@ -1267,4 +1267,82 @@ End Using
 
 {% endhighlight %}
 
+{% endtabs %}  
+
+## Get image rotation angle from OCR processor
+
+For Getting the Image rotation angle, you can rotate the image with 4 angles (0,90,180,360) from OCR Processor. This feature is working in multiple Images and multiple pages. The following code sample illustrates to support to get the Image Rotation angle from OCR Processor.
```

Comment:
To get the Image rotation angle, you can rotate the image with 4 angles (0,90,180, and 360) from the OCR Processor. This feature works in multiple Images and multiple pages. The following code sample illustrates support for the Image Rotation angle from the OCR Processor.